gratipay/utils/fake_data.py

```diff
@@ -309,9 +309,14 @@ def populate_db(db, num_participants=100, ntips=200, num_teams=5, num_transfers=
 
     print("Making Teams")
     teams = []
+    num_teams -= 1      # Make room for the fake Grtipay Team 
     teamowners = random.sample(participants, num_teams)
     for teamowner in teamowners:
         teams.append(fake_team(db, teamowner))
+    
+    # Creating a fake Gratipay Team 
+    teamowner = random.sample(participants, 1)[0]
```

Comment:
I guess the above code in the comment would make this line a little cleaner but it adds more line. I was really trying not to change up to much of the code in too many places.